lms/templates/word_cloud.html

```diff
@@ -1,30 +1,45 @@
+<%page expression_filter="h"/>
 <%! from django.utils.translation import ugettext as _ %>
 
-<section
+<div
     id="word_cloud_${element_id}"
     class="${element_class}"
     data-ajax-url="${ajax_url}"
 >
+    % if display_name is not display_name_default:
+    <h3 class="hd hd-3">${display_name}</h3>
+    % endif
 
-    <section class="input_cloud_section">
+    % if instructions is not instructions_default:
+    <div class="input_cloud_instructions" id="word_cloud_${element_id}_instructions">
+        ${instructions}
+    </div>
+    % endif
+
+    <div class="input_cloud_section" role="group">
```

Comment:
@clrux  This looks like a fieldset/legend situation to me. If you want to emulate a fieldset/legend, the instructions, which serve as the label for the group, would be a child of this div. The div would have an aria-labelledby which referenced the instructions.